### PR TITLE
Bump git to 2.13.2

### PR DESCRIPTION
**Pull request by alarcher**

### CODE REVIEW

**Comment by pyhalov** [comment]
git tests give different results. Likely  results-64.master needs updating.

File: components/developer/git/test/results-64.master [hunk 1]
```diff
@@ -17071,10 +18328,10 @@ ok 66 - prompt - hide if pwd ignored - inside gitdir (stderr)
 # passed all 66 test(s)
 1..66
 
-failed test(s): t0204 t1450 t4062
+failed test(s): t0001 t0204 t1450 t3900 t4062 t8005
```
Comment (pyhalov) [inline components/developer/git/test/results-64.master]: Do you have locale/ja installed?

Comment (pyhalov) [reply components/developer/git/test/results-64.master]: When locale/ja is installed I get:
```
@@ -18328,10 +18930,10 @@
 # passed all 66 test(s)
 1..66

-failed test(s): t0001 t0204 t1450 t3900 t4062 t8005
+failed test(s): t0001 t0204 t1450 t4062

 fixed   1
-success 15478
-failed  9
-broken  196
-total   15920
+success 16028
+failed  4
+broken  198
+total   16473
```

**Review by xen0l — APPROVED** [review]

**Comment by pyhalov** [comment]
Test results are still inconsistent. Tests require library/perl-5/subversion, locale/is, locale/is-extra, library/perl-5/sqlite-dbi.

**Comment by pyhalov** [comment]
also it seems the following lines have random  (or locale-dependent) ordering:
```
@@ -2658,9 +2658,9 @@
 ok 2 - Git::I18N is located at $(@D)/t/../perl/blib/lib/Git/I18N.pm
 ok 3 - sanity: Git::I18N has 3 export(s)
 ok 4 - sanity: Git::I18N exports everything by default
-ok 5 - sanity: N__ has a $ prototype
+ok 5 - sanity: __ has a $ prototype
 ok 6 - sanity: __n has a $$$ prototype
-ok 7 - sanity: __ has a $ prototype
+ok 7 - sanity: N__ has a $ prototype
 ok 8 - Passing a string through __() in the C locale works
 ok 9 - Get singular string through __n() in C locale
 ok 10 - Get plural string through __n() in C locale
```
Overall I get:
```
failed test(s): t0001 t0204 t1450 t4062

fixed   1
success 16033
failed  5
broken  198
total   16473
```
instead of 
```
failed test(s): t0001 t0204 t1450 t4062

fixed   1
success 15497
failed  5
broken  196
total   15935
```

**Comment by alarcher** [comment]
Well still not there.

**Comment by alarcher** [comment]
Oh well... 2.13.2 is already on the way.

**Pull request merged**